apps/prs/src/environments/environment.ts

```diff
@@ -0,0 +1,25 @@
+// This file can be replaced during build by using the `fileReplacements` array.
+// `ng build` replaces `environment.ts` with `environment.prod.ts`.
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
+  firebase: {
+    apiKey: 'AIzaSyD8wq-OjsAkyUf7S2V0Jn2Ofwm-bEx9usM',
+    authDomain: 'test-jperrott.firebaseapp.com',
+    databaseURL: 'https://test-jperrott.firebaseio.com',
+    projectId: 'test-jperrott',
+    storageBucket: 'test-jperrott.appspot.com',
+    messagingSenderId: '61842336122',
+    appId: '1:61842336122:web:b41e86518bcff102',
+  },
+  production: false,
```

Comment:
Just checking: You intentionally want to include the testing instance in Github?

Reply:
Updated to use a different instance for now.